ethportal-api/src/types/content_key/beacon.rs

```diff
@@ -119,20 +195,20 @@ impl OverlayContentKey for BeaconContentKey {
 
         match self {
             BeaconContentKey::LightClientBootstrap(key) => {
-                bytes.push(0x00);
+                bytes.push(0x10);
```

Comment:
```suggestion
                bytes.push(LIGHT_CLIENT_BOOTSTRAP_KEY_PREFIX);
```
I think we should define these as `const`s near the top of the file